keyboards/massdrop/ctrl/keymaps/jmrt/rules.mk

```diff
@@ -0,0 +1,15 @@
+#RGBLIGHT_ENABLE   = no #Not for MD boards.This is here in case you forget.
+COMMAND_ENABLE = no				# Commands for debug and configuration
+#AUTO_SHIFT_ENABLE = yes #Auto Shift
+NKRO_ENABLE = yes           	# USB Nkey Rollover
+DYNAMIC_MACRO_ENABLE = no		# Dynamic macro recording and play
+MOUSEKEY_ENABLE = no			# Enable mouse control keycodes. Increases firmware size.
+TAP_DANCE_ENABLE = no			# Enable tap dance keys
+CONSOLE_ENABLE = no			# Enable debugging console. Increases firmware size.
+SRC += config_led.c				# Used to add files to the compilation/linking list.
+TERMINAL_ENABLE = no
+EXTRAKEY_ENABLE = yes			# Audio control and System control
+#RAW_ENABLE = yes #Raw HID has not yet been implemented for this keyboard
+#COMBO_ENABLE #Key combo                    feature
+#LEADER_ENABLE #Enable leader key           chording
+RGB_MATRIX_CUSTOM_USER = yes
```

Comment:
```suggestion
RGB_MATRIX_CUSTOM_USER = yes

SRC += config_led.c
```

Reply:
Are those lines removed because values provided are default? 
I'm fairly new at this and won't be able to check changes until the weekend or so

Reply:
Yes.